### PR TITLE
move third party js libraries into lib directory

### PR DESCRIPTION
**Pull request by lauraGgit**
Make it clearer which libraries are 3rd party and which are hand rolled.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -10,10 +10,10 @@
 
   <link href="https://fonts.googleapis.com/css?family=Merriweather:400,700,400italic,700italic,300italic,300|Source+Sans+Pro:200,300,300italic,400,400italic,600,600italic,900|Open+Sans+Condensed:300,700" rel="stylesheet">
 
-  <script src="{{site.baseurl}}/javascripts/jquery-2.1.4.min.js"></script>
-  <script src="{{site.baseurl}}/javascripts/application.js"></script>
-  <script src="{{site.baseurl}}/javascripts/tablesaw.stackonly.js"></script>
-  <script src="{{site.baseurl}}/javascripts/tablesaw-init.js"></script>
+  <script src="{{site.baseurl}}/javascripts/lib/jquery-2.1.4.min.js"></script>
+  <script src="{{site.baseurl}}/javascripts/lib/application.js"></script>
```
Comment (andrewmaier) [inline _includes/head.html]: This is all custom code, so I wouldn't put it in the lib :)

**Review by andrewmaier — CHANGES_REQUESTED** [review]

File: _includes/head.html (hunk 1 above)
Comment (andrewmaier) [reply _includes/head.html]: "this" being the application.js

Comment (lauraGgit) [reply _includes/head.html]: oh yes, that makes sense, got trigger happy. Will update.

Comment (lauraGgit) [reply _includes/head.html]: updated by the latest commit.

**Pull request merged**